### PR TITLE
feat: update game speed

### PR DESCRIPTION
**Pull request by zprjd**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (lovebaihezi) [inline .vscode/launch.json:1]: Remove vscode related files

File: crates/game/src/tree.rs [hunk 2]
```diff
@@ -1,14 +1,9 @@
 use bevy::{
-    color::Color,
-    math::{Vec2, Vec3},
-    prelude::{default, Commands, Query, Res, ResMut, Transform},
-    sprite::{Sprite, SpriteBundle},
-    time::{Time, Virtual},
-    window::Window,
+    color::Color, math::{Vec2, Vec3}, prelude::{default, Commands, Query, Res, ResMut, Transform}, sprite::{Sprite, SpriteBundle}, time::{Time, Virtual}, ui::update, window::Window
```
Comment (lovebaihezi) [inline crates/game/src/tree.rs:2]: Run Cargo fmt before commit

File: crates/game/src/tree.rs [hunk 3]
```diff
@@ -47,12 +42,29 @@ pub fn tree_move_animation(
...
         };
     }
 }
+
+
+fn update_game_speed(status: &mut GameStatus) {
+    unsafe {
+        if status.speed < MAX_GAME_SPEED {
+            let new_speed = status.speed.saturating_add(SPEED_INCREMENT);
+            SPEED_INCREMENT = SPEED_INCREMENT.saturating_add(SPEED_INCREMENT);
```
Comment (lovebaihezi) [inline crates/game/src/tree.rs:60]: Speed should not use static mut, consider store it inside component or using bevy resources

File: crates/game/src/tree.rs [hunk 4]
```diff
@@ -47,12 +42,29 @@ pub fn tree_move_animation(
...
+            MAX_GAME_SPEED = MAX_GAME_SPEED.saturating_sub(SPEED_INCREMENT);
+            status.speed = if new_speed >= MAX_GAME_SPEED {
+                MAX_GAME_SPEED
+            } else {
+                new_speed
+            };
+        }
+        println!("tree_move_animation Modified speed: {}  MAX_GAME_SPEED {}", status.speed, MAX_GAME_SPEED);
```
Comment (lovebaihezi) [inline crates/game/src/tree.rs:68]: using function from log, not directly using println!

**Review by lovebaihezi — CHANGES_REQUESTED** [review]

**Comment by lovebaihezi** [comment]
Run `cargo clippy --fix` to fix the error from the CI/CD

**Comment by zprjd** [comment]
> cargo clippy --fix

done

**Review by lovebaihezi — APPROVED** [review]

**Pull request merged**